### ISSUE TITLE
Edit navigation

### ISSUE THREAD

**Issue by vojtech-cerveny** [utituli/vkhcr_web#7]
Navigation is not perfect - add links for mobile devices. 

```
    <nav class="white" role="navigation">
      <div class="nav-wrapper container">
        <a id="logo-container" href="#" class="brand-logo center"><img alt="VKH ČR" src="logos\logo.jpg" width="20%" height="20%"></a>
        <ul class="left hide-on-med-and-down">
          <li><a href="#">O nás</a></li>
          <li><a href="#clenove">Členové</a></li>
          <li><a href="#">Rada</a></li>
        </ul>
        <ul class="right hide-on-med-and-down">
          <li><a href="#">Akce</a></li>
          <li><a href="#">Podpořte nás</a></li>
          <li><a href="#">Kontakt</a></li>
        </ul>

        <ul id="nav-mobile" class="side-nav">
          <!--TODO add links here too  -->
          <li><a href="#">O nás</a></li>
        </ul>
        <a href="#" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">menu</i></a>
      </div>
    </nav>
```

**Comment by utituli** [comment]
I know about it. I will copy - paste all after web template is finished.

**Comment by vojtech-cerveny** [comment]
Fixed in #17
